libcodechecker/analyze/plist_parser.py

```diff
@@ -102,8 +103,12 @@ def parse_plist(path):
 
             # We need to extend information for plist files generated
             # by older clang version (before 3.8).
+            file_path = files[diag['location']['file']]
+            if source_root is not None:
```

Comment:
`if x is not None` can be replaced with `if x` if `source_root` isn't a boolean.